src/main.rs

```diff
@@ -1,5 +1,20 @@
 use std::{cmp, env, fs};
 
+/// The program gets the two files it will process from the command line.
+/// Then it applies the longest_common_subsequence function. This function creates a table of
+/// comparisons that eventually gives print_diff a path to follow when printing. While following
+/// this path, print_diff will let us know how the two files differ and coincide. This is shown
```

Comment:
It should say "print_diff will let us know how the two files differ and coincide _from each other_".

Reply:
Corrected in 32b6ed08865fd44833198f3ab995ffe8612ef1aa